File: apps/websocket-skyblock-csharp/src/mongodb/collections/BazaarItemsAll.cs
using System.Runtime.Serialization;
using MongoDB.Bson.Serialization.Attributes;

public class BazaarItemsAll
{
    [BsonId]
    [DataMember]
    public MongoDB.Bson.ObjectId _id { get; set; }
    /// <summary>
    /// Hypixel ID of the item.
    /// </summary>
    public string ID = "";
    /// <summary>
    /// Name of the item.
    /// </summary>
    public string Name = "";

    public BazaarItemsAll(string _ID, string _Name)
    {
        ID = _ID;
        Name = _Name;
    }
}

File: apps/discord-bot/app/src/main/java/discordbot/mongodb/collections/BazaarItemsAll.java
package discordbot.mongodb.collections;

import org.bson.codecs.pojo.annotations.BsonId;
import org.bson.types.ObjectId;

public class BazaarItemsAll {
    @BsonId
    public ObjectId _id;
    /**
     * The Hypixel ID of the item.
     */
    public String ID;
    /**
     * The name of the item.
     */
    public String Name;

    public BazaarItemsAll(String _ID, String _Name) {
        ID = _ID;
        Name = _Name;
    }
}
